game/scripts/npc/items/reflex_cores/react/item_reactive_3a.txt

```diff
@@ -35,7 +35,7 @@
 		"AbilityTextureName"    "custom/item_reactive_3a"
 		// Stats
 		//-------------------------------------------------------------------------------------------------------------
-		"AbilityCooldown"       "10.0"
+		"AbilityCooldown"       "0.5"
```

Comment:
Reactive 3 : Grants 0,5s invulnerability and throws blocked spells back, gains 3 regenerating charges

Giving this a 0.5 second CD huh? Sniffing too many sharpies are we?